### PR TITLE
PRESIDECMS-1885 grid listing header label improvements

### PR DESCRIPTION
**Pull request by pixl8-brayden**
- add new proxies helper to get base uri root
- add new context argument for i18n$translatePropertyName
- add optional tooltip (help) for data table listing header
i18n file sample
data table listing sample

### CODE REVIEW

File: system/views/admin/datamanager/_objectDataTable.cfm [hunk 1]
```diff
@@ -171,7 +171,15 @@
 						</th>
 					</cfif>
 					<cfloop array="#args.gridFields#" index="fieldName">
-						<th data-field="#ListLast( fieldName, '.' )#">#translatePropertyName( args.objectName, fieldName )#</th>
+						<th data-field="#ListLast( fieldName, '.' )#">
+							#translatePropertyName( args.objectName, fieldName )#
```
Comment (DominicWatson) [inline system/views/admin/datamanager/_objectDataTable.cfm]: Should this have a context of 'listing', i.e.

```translatePropertyName( args.objectName, fieldName, "listing" )```

**Review by DominicWatson — COMMENTED** [review]
Looks good, I just have a query about the implementation in the listing view. Could you check that?

Also, could you find somewhere to add to the documentation? :) I think somewhere in the datamanager section would make sense.

File: system/views/admin/datamanager/_objectDataTable.cfm (hunk 1 above)
Comment (pixl8-brayden) [reply system/views/admin/datamanager/_objectDataTable.cfm]: thanks for notice this, my bad was missed out when copying over.

**Comment by DominicWatson** [comment]
Fab, thanks @pixl8-brayden - merging into 10.12.0

**Pull request merged**